### ISSUE TITLE
Search box disappears as soon as I start typing something

### ISSUE THREAD

**Issue by jaytrinitron** [Florry/spicetify-playlist-filter#5]
Hello, when I enable this extension from marketplace, the search box disappears as soon as I start typing something (even if I am not pressing "f"), and pressing "f" does not bring it back. I assume one of the other extensions is interfering with the behavior but I'm not sure what it is. I'm only using spicetify-cli and spicetify marketplace enhancements. Perhaps the problem could also be a new Spotify update. 

Also, this is less of an issue because I do not need the playlist filter to have a keyboard shortcut, but it would be nice if we could change the keyboard shortcut from "f" since some of the other extensions also use this key.

**Comment by Florry** [comment]
Hm, I don't think it's the new update of spotify that is the problem, unless they did some more updates recently that broke it; I did my latest fixes on the new spotify update, but my client has been "updated" to the old version since then so I can't really know for sure at this moment 🤔Would you mind providing a list of the spicetify marketplace enhancements you have installed so I can do some debugging and see if I find any interference, and if so, fix it? 

As for the keyboard shortcut customization, that's a great suggestion. I will add it 🙂

**Comment by jaytrinitron** [comment]
Thank you!

>Would you mind providing a list of the spicetify marketplace enhancements you have installed so I can do some debugging and see if I find any interference, and if so, fix it?

Sure! I'm not sure how to "list" the marketplace ones exactly though; hopefully pictures are acceptable and will save both of us time since I can't copy and paste. If you know of a better way to list them please let me know.

![image](https://user-images.githubusercontent.com/62568097/197270667-eac384b0-a123-41fa-8a2a-23f76dfb37d6.png)
![image](https://user-images.githubusercontent.com/62568097/197270787-f3f7b877-8dec-4031-ba82-c67498773abc.png)

And in spicetify-cli I am also using these in the config file:
extensions            = autoSkipVideo.js|bookmark.js|keyboardShortcut.js|loopyLoop.js|popupLyrics.js|trashbin.js
custom_apps        = marketplace|reddit|lyrics-plus

I tried a couple different themes and they did not seem to have an effect. Also tried removing some of the playlist sidebar related extensions and snippets, but to no avail.

**Comment by Florry** [comment]
That's perfect, thanks a lot! I will let you know if I find anything 🙂

**Comment by Florry** [comment]
Sorry I haven't gotten around to doing the debugging yet. However, I recently saw the star rating extension being reported not to work with the new spotify update, but then the new spicetify update released yesterday fixed it. Would you mind upgrading your spicetify and testing the playlist-filter with it and see if the problem still persists; my spotify is still stuck on the old client? https://github.com/spicetify/spicetify-cli/releases/tag/v2.14.1

**Comment by jaytrinitron** [comment]
> Sorry I haven't gotten around to doing the debugging yet. 

No worries, take your time.

> Would you mind upgrading your spicetify and testing the playlist-filter with it and see if the problem still persists?

I already did and it does persist.

**Comment by Florry** [comment]
Alright, thanks for testing!

**Comment by Florry** [comment]
Good news! I started getting this issue myself yesterday so I was able to locate what was causing it and fix it, hopefully it is the same issue you are having. So the problem is that the response from the Spotify backend has started including a new type `placeholder` (whatever that is) and I was naively treating everything not being a folder as a playlist, so it crashed. See https://github.com/Florry/spicetify-playlist-filter/commit/2c52d85274a83eca402334338462a2da8f423c36

It took my spicetify marketplace +16 hours to actually grab the latest changes when installing through it, so it might not be available right away for you, not sure how the caching works there but let me know if it works 🙂
